driver/pace/driver/driver.py

```diff
@@ -280,6 +280,8 @@ def from_dict(cls, kwargs: Dict[str, Any]) -> "DriverConfig":
             # Copy grid_type to the DycoreConfig if it's not the default value
             if grid_type != 0:
                 kwargs["dycore_config"].grid_type = grid_type
+                if grid_type > 3:
```

Comment:
I am hoping that in the future we can use a better coding for grid type than just integers. Python's datatypes allow more flexibility than do Fortran integers.

Reply:
It is logged: https://github.com/NOAA-GFDL/pace/issues/29

Reply:
Yeah that's absolutely in the cards